nl_server/__init__.py

```diff
@@ -28,6 +28,22 @@ def create_app():
 
   flask_env = os.environ.get('FLASK_ENV')
 
+  # Download existing embeddings (if not already downloaded).
+  models_downloaded_paths = {}
```

Comment:
models_downloaded_paths is not used in this module?

Reply:
Good point. Made the changes